src/Client/Components/EntityTable/AddEditModal.razor.cs

```diff
@@ -4,6 +4,7 @@
 using MudBlazor;
 
 namespace FSH.BlazorWebAssembly.Client.Components.EntityTable;
+public partial class AddEditModal{}
```

Comment:
Why another partial?

Reply:
https://stackoverflow.com/questions/42647384/asp-net-core-localization-with-help-of-sharedresources

Please check this. For such cases, instead of using partials and stuff, we can simply create a sharedResource

Reply:
added shared resource for manage common strings